client/app/account/billing/common/renew-helper.service.js

```diff
@@ -31,9 +31,10 @@ angular
             }
 
             if (angular.isObject(service)) {
-                if (service.serviceType === "EMAIL_DOMAIN") {
+                if (service.serviceType === "EMAIL_DOMAIN" || service.serviceType === "SMS") {
```

Comment:
Could also be written in that manner:
```js
_.includes(["EMAIL_DOMAIN", "SMS"], service.serviceType);
```

Reply:
I like your way better as it is easier to add new service types but I didn't do it as the page will be completely redone soon